configurations/example.yaml

```diff
@@ -1,6 +1,6 @@
 # File metadata, to be printed by the simulator
 name:
-  Random Cases, Preset Bases, Preset Travel Times
+  "Random Cases, Preset Bases, Preset Travel Times"
```

Comment:
Is there a reason why these are in a string?

Reply:
YAML does not accept @ symbols (which we needed for referencing objects) unless it's explicitly a string. 

e.g. @object_tag must change to "@object_tag"

So I went ahead and added quotations to all strings.